logos/logos-workernode/logos_worker_node/lane_manager.py

```
@@ -1398,17 +1408,47 @@ async def _check_stuck_lanes(self, statuses: list[LaneStatus]) -> None:
                                        details=f"gen_tokens={gen_tokens}, running={requests_running}, polls={count}")
                     self._stuck_polls.pop(lid, None)
                     self._last_gen_tokens.pop(lid, None)
-                    # Kill the stuck lane (outside the lock since stop acquires it)
+                    # Kill the stuck lane and attempt automatic restart
                     handle = self._handles.get(lid)
                     if handle is not None:
+                        lane_config = handle.lane_config
                         try:
                             await handle.stop()
                             logger.info("Lane '%s' stopped after stuck detection", lid)
                         except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)